.github/workflows/release.yml

```
@@ -0,0 +1,125 @@
+name: Create Zeek release
+
+on:
+  workflow_dispatch:
+    inputs:
+      zeek_ref:
+        required: false
+        type: string
+        default: 'master'
+      build_zeek_ref:
+        required: false
+        type: string
+        default: 'main'
+      release_tag:
+        required: true
+        type: string
```

Any reason not to follow the pattern we use elsewhere, with dependencies pinned explicitly here in the repository (for Zeek, that'd be in the "Checkout zeek" step) and release creation triggered by pushing a tag?

(The advantage of that pattern is that checking out a release tag gets you exactly what was used to build the release.)

```suggestion
on:
  pull_request:
  push:
    branches:
      - master
    tags:
      - v*brim*
  workflow_dispatch:
```